InteractiveRunner.html

```diff
@@ -1,22 +1,39 @@
 <!DOCTYPE html>
 <html>
-<head>
-<title>Speedometer 2.1 Interactive Runner</title>
-<script src="resources/interactive.mjs" type="module"></script>
-<style>
-iframe { border: 1px solid black; }
-ol { list-style: none; margin: 0; padding: 0; }
-ol ol { margin-left: 2em; list-style-position: outside; }
-.running { text-decoration: underline; }
-.ran { color: grey; }
-nav { position: absolute; right: 10px; height: 600px; }
-nav > ol { height: 100%; overflow-y: scroll; }
-</style>
-</head>
-<body>
-<script>
-
-
-</script>
-</body>
+    <head>
+        <title>Speedometer 2.1 Interactive Runner</title>
+        <script src="resources/interactive.mjs" type="module"></script>
+        <style>
+            iframe {
+                border: 1px solid black;
+            }
+            ol {
+                list-style: none;
+                margin: 0;
+                padding: 0;
+            }
+            ol ol {
+                margin-left: 2em;
+                list-style-position: outside;
+            }
+            .running {
+                text-decoration: underline;
+            }
+            .ran {
+                color: grey;
+            }
+            nav {
+                position: absolute;
+                right: 10px;
+                height: 600px;
+            }
+            nav > ol {
+                height: 100%;
+                overflow-y: scroll;
+            }
+        </style>
+    </head>
+    <body>
+        <script></script>
```

Comment:
I guess we can get rid of this empty script element? Can be done in a separate PR though.

Reply:
might as well do it now 😄